frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -121,6 +124,15 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
     [setPageSize, setPageIndex],
   );
 
+  // Update internal state whenever the filter bar state changes
+  const handleFilterChange = useCallback(
+    async (value: IProjectFilterState) => {
+      setFilter({ ...value });
+      setPageIndex(0); // Go to first page of results when filter changes
+    },
+    [setFilter, setPageIndex],
+  );
+
```

Comment:
`async` is usually combined with a call to `await` in the code but I can't find any - perhaps remove `async` here?